centreon/www/class/centreonACL.class.php

```diff
@@ -1447,23 +1447,26 @@ public function queryBuilder($condition, $field, $stringlist)
      *
      * @param string $p
      * @param bool $checkAction
+     *
      * @return int | 1 : if user is allowed to access the page
      *               0 : if user is NOT allowed to access the page
      */
-    public function page($p, $checkAction = false)
+    public function page(string $p, bool $checkAction = false)
     {
         $this->checkUpdateACL();
         if ($this->admin) {
-            return 1;
+            return self::ACL_ACCESS_READ_WRITE;
         } elseif (isset($this->topology[$p])) {
-            if ($checkAction && $this->topology[$p] == 2 &&
-                isset($_REQUEST['o']) && $_REQUEST['o'] == 'a'
+            if (
+                $checkAction
+                && $this->topology[$p] == self::ACL_ACCESS_READ_ONLY
+                && isset($_REQUEST['o']) && $_REQUEST['o'] == 'a'
```

Comment:
Strictness + idea of writing for lisibility (subjective).
```suggestion
                && ($_REQUEST['o'] ?? null) === 'a'
```